### PR TITLE
Prerequisite and a Note added for macOS 

### PR DESCRIPTION
**Pull request by fmcdonal**
Added a prerequisite and a NOte for macOS in the Enabling command completion on Bash

### CODE REVIEW

**Comment by fmcdonal** [comment]
mmurphy@redhat.com
pls review

File: docs/rhoas/rhoas-cli-installation/README.adoc [hunk 1]
```diff
@@ -230,6 +230,7 @@ You can enable command completion for each of the following shells:
 .Prerequisites
 
 * You've installed the `rhoas` CLI.
+* To enable command completion on Bash for macOS you need to have Bash completion installed and running with its current login scripts.
```
Comment (bhardesty) [inline docs/rhoas/rhoas-cli-installation/README.adoc:233]:
```suggestion
* If you're using macOS, `bash-completion` must be installed and configured in your `.bash_profile` file.
```

File: docs/rhoas/rhoas-cli-installation/README.adoc [hunk 2]
```diff
@@ -257,6 +258,9 @@ $ sudo mv rhoas_completions /etc/bash_completion.d/rhoas
 ----
 $ sudo mv rhoas_completions /usr/local/etc/bash_completion.d/rhoas
 ----
+
+NOTE: For macOS consult your bash software for the location and usage of your bash_completion directory.
```
Comment (bhardesty) [inline docs/rhoas/rhoas-cli-installation/README.adoc:262]: If the user's bash-completion is in a different location, I believe they can find it by checking .bashrc or .bash_profile in their home directory:

```suggestion
NOTE: For macOS, to find the location of your `bash-completion` directory, check the `~/.bashrc` or `~/.bash_profile` file.
```

File: docs/rhoas/rhoas-cli-installation/README.adoc (hunk 1 above)
Comment (jbyrne-redhat) [reply docs/rhoas/rhoas-cli-installation/README.adoc:233]: Just one further suggestion here. I would try to write this in the active voice, to match the prerequisite that comes before it. So, something like:

"If you're using macOS, you've installed `bash-completion` and configured this in your `.bash_profile` file."

**Comment by jbyrne-redhat** [comment]
@fmcdonal I think you're adding a file called `CONTRIBUTING.HTML` that you didn't intend to. This likely happened when you opened `CONTRIBUTING.adoc` and then generated (and saved) a preview.

You can remove this file in your local repo by running `git rm CONTRIBUTING.html` (in the root `app-services-guides` directory) and committing the change. Pushing that fresh commit will remove the file from this PR.

File: docs/rhoas/rhoas-cli-installation/README.adoc (hunk 2 above)
Comment (fmcdonal) [reply docs/rhoas/rhoas-cli-installation/README.adoc:262]: Implemented, thanks

**Review by bhardesty — APPROVED** [review]

**Pull request merged**